en/docs/administer/customize-the-developer-portal.md

```diff
@@ -5,17 +5,17 @@ The Developer Portal allows API consumers to find and consume APIs with ease. Yo
 To customize the Developer Portal theme, follow the steps given below:
 
 !!! Note
-    - To customize the Developer Portal theme for an organization, you need to be an admin user of that organization.<br/><br/>
+    - To customize the Developer Portal theme for an organization, you need to be an admin user of that organization.
     - You cannot undo a change and restore or revert to a previous version of the theme. However, you can reset it to the default theme.
 
-1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/) using a Google, GitHub, or Microsoft account.
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/) using a Google/ GitHub/ Microsoft account.
```

Comment:
Seems like the commas have been mistakenly replaced with /.

Reply:
This was used in many other docs, hence kept it this way for consistency.